data_adapter_oemof/adapters.py

```diff
@@ -14,34 +11,18 @@
 
 class Adapter:
     type: str = "adapter"
+    facade: Facade = None
     extra_attributes = ("name", "type")
 
-    def as_dict(self):
-        """
-        Adds function to return DataFrame from adapter.
-
-        This mixin is needed because `pd.DataFrame(dataclass_instance)` creates columns
-         only for existing dataclass attributes.
-        But we add custom_attributes (i.e. "name") which would be neglected.
-        """
-        fields = dataclasses.fields(self)
-        data = {field.name: getattr(self, field.name) for field in fields}
-        for attr in self.extra_attributes:
-            data[attr] = getattr(self, attr)
-        return data
-
-    @classmethod
-    def parametrize_dataclass(
-        cls,
-        struct,
-        mapper: Mapper,
-    ) -> "Adapter":
-        return cls(**cls.get_default_parameters(struct, mapper))
-
-    @classmethod
-    def get_default_parameters(cls, struct: dict, mapper: Mapper) -> dict:
+    def as_dict(self) -> dict:
+        return self.facade_dict
```

Comment:
Why do you want to keep the `as_dict` attribute? 
To me this appears a little confusing and like an extra step to just change the name.

Reply:
only synthetic sugar. but i will remove it